CaptureSDK/GiniCaptureSDK/Documentation/source/Customization guide.md

```diff
@@ -70,17 +70,7 @@ Images customization is done via overriding of [GiniImages.xcassets](https://git
 - Error background &#8594;  `GiniConfiguration.noticeErrorBackgroundColor`
 - Error text color `GiniConfiguration.noticeErrorTextColor`
 
-##### 3. Tooltip
-<center><img src="img/Customization guide/Tooltip.jpg" height="120"/></center>
-- Background color &#8594;  `GiniConfiguration.fileImportToolTipBackgroundColor`
-- Text color &#8594;  `GiniConfiguration.fileImportToolTipTextColor`
-- Close button color &#8594;  `GiniConfiguration.fileImportToolTipCloseButtonColor`
-- Text &#8594; 
-    - <span style="color:#009EDF">*ginicapture.camera.fileImportTip*</span> localized string for file import tooltip
-    - <span style="color:#009EDF">*ginicapture.camera.qrCodeTip*</span> localized string for qr code tooltip
-    - <span style="color:#009EDF">*ginicapture.multipagereview.reorderContainerTooltipMessage*</span> localized string for reorder tooltip
-
-##### 4. Gini Capture font
+##### 3. Gini Capture font
```

Comment:
`Gini Capture font` will be removed as well. I've already merged customization for the Camera screen - can we make sure that this version will be based on the version in the new UI now? 😊

Reply:
@vizaknaidavid  I've seen the merge conflict you can just use the version from new ui  for Customization guide.md